app/bundles/CoreBundle/Helper/IpLookupHelper.php

```diff
@@ -139,6 +145,9 @@ public function getIpAddress($ip = null)
 
             if ($ipAddress === null) {
                 $ipAddress = new IpAddress();
+                if ($this->coreParametersHelper->getParameter('anonymize_ip')) {
+                    $ip = preg_replace('/(?!\d{1,3}\.\d{1,3}\.)\d/', '*', $ip);
```

Comment:
Are you aware of existence of IPv6 addresses ?
This will convert IPv6 address `fe80::3888:2b25:2f3a:b35b` to `fe**::****:*b**:*f*a:b**b` and i dont think that is correct result...
see https://regex101.com/r/uj7cuA/1

Reply:
Improved regexp based on  https://stackoverflow.com/a/48777412